recipe/meta.yaml

```diff
@@ -1,16 +1,16 @@
 {% set name = "crc32c" %}
-{% set version = "1.1.1" %}
+{% set version = "1.1.2" %}
 
 package:
   name: lib{{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/google/{{ name }}/archive/{{ version }}.tar.gz
-  sha256: a6533f45b1670b5d59b38a514d82b09c6fb70cc1050467220216335e873074e8
+  sha256: ac07840513072b7fcebda6e821068aa04889018f24e10e46181068fb214d7e56
 
 build:
-  number: 2
+  number: 1
```

Comment:
as you raised version, the build number should be reset for it to zero

Reply:
additionally the run_export part here should go away

Reply:
I was horribly mistaken.